tests/lib/snaps/test-snapd-removable-media/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-removable-media
+version: 1.0
+summary: Basic removable-media snap
+description: A basic snap which access to removable storage filesystems
+
+apps:
+  sh:
+    command: ../../../bin/sh
```

Comment:
Ouch! This ought to be invalid. It should be an absolute path instead.

@chipaca Did we get that fixed?  I recall we talked about it recently, but don't know if we acted on it.